doc/standards-object.md

```diff
@@ -39,13 +39,13 @@ The [`ariaAttrs`](../lib/standards/aria-attrs.js) object defines valid ARIA attr
 - `type` - string(required). The attribute type which dictates the valid values of the attribute. Valid types are:
   - `boolean` - Boolean attributes only accept `true` or `false` as valid values (e.g. `aria-selected`).
   - `nmtoken` - Name token attributes accept a single value from a list of valid values (e.g. `aria-orientation`).
-  - `mntokens` - Name tokens attributes accept a space separated list of values from a list of valid values (e.g. `aria-relevant`).
```

Comment:
Let's file a separate issue/PR for correcting this (here and on L48); this will also need a corresponding update to our typings and tests.